frontend/src/components/Header.tsx

```
@@ -0,0 +1,68 @@
+    ':hover': {
+        backgroundColor: alpha(theme.palette.common.black, 0.05),
+    },
+}));
+
+const SearchInputStyled = styled(InputBase)(() => ({
+    color: 'black',
+    fontSize: 'small',
+    backgroundColor: 'transparent',
+  }));
+
+export default function Header({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)